tools/CoqMakefile.in

```diff
@@ -162,6 +161,31 @@ ifdef DSTROOT
 DESTDIR := $(DSTROOT)
 endif
 
+# findlib files installation
+ifeq ($(DESTDIR),)
+FINDLIBPREINST= true
+FINDLIBDESTDIR= -destdir $(COQCORELIB)../
+else
+FINDLIBPREINST= mkdir -p $(DESTDIR)/$(COQCORELIB)../
+FINDLIBDESTDIR= -destdir $(DESTDIR)/$(COQCORELIB)../
+endif
+findlib_install = \
+	$(HIDE)if [ -e META ]; then \
+	  $(FINDLIBPREINST) && \
+	  mv $(METAFILE) $(METAFILE).skip ; \
+	  $(OCAMLFIND) install $(2) $(FINDLIBDESTDIR) $(FINDLIBPACKAGE) $(1); \
+	  rc=$$?; \
+	  mv $(METAFILE).skip $(METAFILE); \
+	  rm -f META; \
```

Comment:
The META manipulations are kinda confusing, eg why are we deleting it here?
Also when METAFILE is empty the rule to build META doesn't actually build it, this is strange.

Reply:
ok, I'll try to rephrase things.